apis/core/bootstrap/users.ts

```diff
@@ -0,0 +1,12 @@
+import { turtle } from '@tpluscode/rdf-string'
+import { hydra, rdf, schema } from '@tpluscode/rdf-ns-builders'
+
+export const users = turtle`
+<users> {
+  <users> a ${hydra.Collection} ;
+    ${hydra.manages} [
+      ${hydra.property} ${rdf.type} ;
+      ${hydra.object} ${schema.Person}
+    ] .
+}
+`
```

Comment:
This reflects a bigger change where users actually become independent resources in their own named graphs. Here, the collection is added to retrieve them

Reply:
I love this change!